src/mime/media_type.cr

```diff
@@ -5,8 +5,7 @@ require "mime"
 module MIME
   # A MediaType describes a MIME content type with optional parameters.
   struct MediaType
-    property media_type : String
-    property params : Hash(String, String)
```

Comment:
Why? Having access to raw params Hash could come handy, when removed there's no way to do that.

Reply:
`MediaType` directly exposes `#[]`, `#[]?`, `#fetch`, `#[]=` and `#each_parameter`. Why would yo need direct access to `params`?

Reply:
For serialization as an example.

Reply:
What kind of serialization? I figure you'd either serialize with `#to_s` or directly access the ivar `@params` (like `JSON::Serializable`).